pkgs/mike-engel/jwt-cli/registry.yaml

```diff
@@ -8,19 +8,14 @@ packages:
       - name: jwt
     version_constraint: "false"
     version_overrides:
-      - version_constraint: semver("<= 0.2.0")
-      - version_constraint: Version == "0.3.0"
-        asset: jwt-cli-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
-        format: zip
-        windows_arm_emulation: true
-        replacements:
-          amd64: x86_64
-          windows: pc-windows-msvc
-        supported_envs:
-          - windows/amd64
```

Comment:
Why did you remove this?

```console
$ cmdx lsa -r mike-engel/jwt-cli 0.3.0
+ REPO=${REPO#https://github.com/}
repo=$(bash scripts/get_test_pkg.sh "$REPO")

gh release view --json assets --jq ".assets[].name" -R "$repo" "$VERSION"

jwt-cli-0.3.0-x86_64-pc-windows-msvc.zip
```

Reply:
Hmm.
The release date is very old. ` Mar 8, 2017`

https://github.com/mike-engel/jwt-cli/releases/tag/0.3.0

Okay. I think we can remove this.

Reply:
Yeah, I removed it because I had trouble getting it to work, and it was not supported before this, either.
https://github.com/aquaproj/aqua-registry/pull/31417/files#diff-b702041437ded01ae13c680eacd08b8b1fe99ef702920420435f8c5323aac289L137-L138

Reply:
Oh, I removed this before intentionally, but I don't remember the reason.

- https://github.com/aquaproj/aqua-registry/pull/15582/commits/e3b21494f7577eb2b2e4e813f808a47416b18b73